docs/blog/posts/2024/week33.md

```diff
@@ -0,0 +1,81 @@
+| `@warp-ds/vue`      | `2.1.0`        |
+| `@warp-ds/elements` | `2.0.0`        |
+| `@warp-ds/icons`    | `2.1.0-next.2` |
+
+If you have already followed the [previous instructions for the DBA beta changes](https://warp-ds.github.io/tech-docs/blog/posts/2024/warp-2-0) and have disabled EIK import mapping for `react`, `vue` & `elements`, you can now re-enable it.
+
+For further instructions on how to migrate from v1 to v2, please follow [these instructions](https://warp-ds.github.io/tech-docs/blog/posts/2024/warp-2-0#migrating-from-v1-to-v2).
+
+## Breaking changes
+### @warp-ds/css 2.0.0
+[Changelog](https://github.com/warp-ds/css/releases/tag/v2.0.0)
+
+The following exported objects/classes have been removed:
+
+| Exported object            | Class name(s)                                                            | Note                                                          |
+|----------------------------|--------------------------------------------------------------------------|---------------------------------------------------------------|
+| backwardsCompatibleClasses |                                                                          | Entire exported object has been removed                       |
+| badge                      | `borderedClickable`, `infoClickable`, `neutralClickable`, `notification` | Removed                                                       |
+| card                       | `cardUnselected`                                                         | Removed                                                       |
+| expandable                 | `chevronBox`, `expandableInfo`                                           | Removed                                                       |
+| helpText                   | `helpTextValid`, `helpTextInvalid`                                       | Removed                                                       |
+| label                      | `labelInvalid`                                                           | Removed                                                       |
+| modal                      | `transparentBg`                                                          | Removed                                                       |
+| ribbon                     |                                                                          | Entire exported object has been removed and replaced by badge |
+| switchToggle               | `switch`                                                                 | Removed                                                       |
+
+
+
+### @warp-ds/react 2.0.0
+[Changelog](https://github.com/warp-ds/react/releases/tag/v2.0.0)
+
+The following properties have been removed:
+
+| Component  | Property               | Note                              |
+|------------|------------------------|-----------------------------------|
+| Box        | Clickable              | Removed                           |
```

Comment:
```suggestion
| Box        | clickable              | Removed                           |
```

Could we use lowercase here? I just thought that `Clickable` might be mistaken with a corresponding component :)